johnzon-jsonb/src/main/java/org/apache/johnzon/jsonb/JohnzonBuilder.java

```diff
@@ -131,9 +131,12 @@ public Jsonb build() {
                     return it;
                 }).orElse(false);
 
-        if (config.getProperty(JsonbConfig.FORMATTING).map(Boolean.class::cast).orElse(false)) {
-            builder.setPretty(true);
-        }
+        config.getProperty(JsonbConfig.FORMATTING).map(Boolean.class::cast).ifPresent(builder::setPretty);
+        config.getProperty(AbstractJsonFactory.BUFFER_STRATEGY).map(String::valueOf).ifPresent(builder::setBufferStrategy);
+        config.getProperty(JsonParserFactoryImpl.BUFFER_LENGTH).map(Integer.class::cast).ifPresent(builder::setBufferSize);
+        config.getProperty(JsonParserFactoryImpl.MAX_STRING_LENGTH).map(Integer.class::cast).ifPresent(builder::setMaxSize);
+        config.getProperty(JsonParserFactoryImpl.SUPPORTS_COMMENTS).map(Boolean.class::cast).ifPresent(builder::setSupportsComments);
+        config.getProperty(JsonParserFactoryImpl.ENCODING).map(String::valueOf).ifPresent(builder::setEncoding);
```

Comment:
Must use jsonb property, not jsonp one

Reply:
Completely oversaw that I used the jsonp property here for the encoding, thanks!
You however marked 4 lines with this comment, all the other properties look very specific to the json-p impl. Am I missing something here?

Reply:
Johnzon properties are all renamed with shorter aliases in jsonb binding ("johnzon.x"). Jsonp properties are often way too long so think it is worth aligning it on this convention.

Reply:
Sure thing, but I wouldn't include this change in this PR, this is only about JOHNZON-319 afterall.
Maybe settle with the full jsonp properties for now, and change the properties to jsonb style ("johnzon.*") later on? What do you think?

Reply:
Hmm, first thought was why merging something we dont want and which does not match the ticket (if you map all mapper props you map them johnzon.x style) but I can miss sthg (reviewed from the phone this time).

Reply:
I just don't think this PR should also contain the aliasing of the jsonp properties to jsonb-style properties. But if you think otherwise I'll include it.

Reply:
Hmm, any PR must not use as jsonb properties some jsonp properties by design or it should use a filter like + mapping algorithm (prefix or so) to create a kind of passthrough mode for properties. Aliases are current way to code it, subnamespace would work too. Jsonb module must also not include johnzon-core imports. in these regards this PR is not mergeable since it creates a naming inconsistency and breaks module design, this is why I mentionned it.

Thinking out loud i wouldnt map all properties for jsonp but a single "properties" (map) one and wire it in all jsonb config consumers (jaxrs providers for ex).